docs/plugins/repository-azure.asciidoc

```diff
@@ -82,13 +85,15 @@ cloud:
                 account: your_azure_storage_account1
                 key: your_azure_storage_key1
                 default: true
+                max_retries: 7
             my_account2:
                 account: your_azure_storage_account2
                 key: your_azure_storage_key2
                 timeout: 30s
 ----
 
-In this example, timeout will be 10s for `my_account1` and 30s for `my_account2`.
+In this example, timeout will be 10s for `my_account1` with 7 retries before failing
```

Comment:
Sure. The timeout interval resets with each retry, if the request is retried. Which means that we will fail after 10 seconds here.
Then we will wait for 30s (the initial backoff period) and try again. After 10 seconds, we will try again after a given period (30s + x)... And so on.
Makes sense? I'm going to reword that part.